app/mailers/notification_mailer.rb

```diff
@@ -11,6 +11,11 @@ def daily_summary_email(addressee, summary_report = nil)
     mail(to: addressee.email, subject: t('.subject', date: Date.yesterday))
   end
 
+  def daily_auctions_broadcast_email(recipient:, auctions:)
+    @auctions = auctions
+    mail(to: recipient.email, subject: t('.subject'))
```

Comment:
Subject is localized by default. Better to remove it, and instead use translation files when needed.